escape-room/doc/research.md

```diff
@@ -0,0 +1,59 @@
+---
+title: "Recherche zu virtuellen Escape-Rooms in der Lehre"
+---
+
+# Arbeitsschritte
+
+## Allgemeine Recherche
+
+- Wie funktionieren Escape-Rooms?
+  - [Design von Escape-Rooms](research_design.md)
+  - [Erzählung der Geschichte](research_narrative.md)
+  - [verwandte Lösungen in der Wissenschaft](./research_solutions_science.md)
+  - [andere Lösungen](./research_solutions_other.md)
+- Was sind wiederkehrende Spielelemente?
+  - „Funktioniert gut digital“
+  - „Funktioniert digital eher nicht“
+  - Sammlung und Aufarbeitung in Miro
+  - Ergebnis anschließend hier versionieren
+- Daraus eine Art Featureliste generieren
+
+## Erstellung des analogen Escape-Rooms & Durchführung der Evaluierung
+
+- Lerninhalte definieren (das habt ihr ja mit Git irgendwie schon festgelegt) und die Features darauf anwenden
+- Zweck der Durchführung konkretisieren
+- Frage klären: Wie evaluieren wir das Ganze?
+- Fertig basteln, Studis spielen lassen, auswerten
+
+## Anwendung des Gelernten auf die Planung der kommenden Schritte
+
+- erste Versionen von Vorgehen, Features und Evaluierungskonzepte anlegen
+
+# Einordnung des Escape-Rooms für den Dungeon
+
+## Begriffsklärung
+
+Die grundlegende Spielidee (Ausbruch aus einem begrenzten Raum) für Escape-Rooms geht auf ein Textbuch-Adventure (Interactive Fiction) aus 1988 zurück. Bis in die frühen 2000er werden diverse auf dem Konzept aufbauende "Point-And-Click" Videospiele und digitale Text-Adventures veröffentlicht, die den Begriff "Escape-Room" / "Escape Game" zunächst prägen. In dieser Zeit werden die ersten direkt von diesen Spielen inspirierten, physischen Räume konstruiert, die auch für mehrere Spieler gedacht sind. Ab 2015 wächst das Interesse der breiteren Öffentlichkeit in das Genre "Escape Room", das sich vornehmlich auf die physischen Räume bezieht, nachdem kurze Zeit vorher in China das artverwandte "Jubensha" an Popularität gewonnen hat (Murder-Mystery-Rollenspiel im Team).
+
+Im Kontext von Videospielen sind 'Escape-Rooms' ein Subgenre von Point-And-Click-Adventures oder Text-Adventures, die prinzipiell alleine und ohne Zeitdruck gespielt werden. Die Ästhetik dort ist nicht selten eine "zwanghafte Neugier", die von Befremdlichkeit oder auch Horror genährt wird. In der Öffentlichkeit sind "Escape Rooms" gemeinschaftliche Aktivitäten in gemieteten Räumen mit einem eher durchgeplanten Spielverlauf und einem (zunächst der Vermarktung geschuldeten) Zeitlimit.
+
+## Einordnung in das Forschungsprojekt und den Dungeon
```

Comment:
Das erklärt irgendwie das Projekt.
Sollte vermutlich in die top level readme des `escape-room/doc`